lib/delayed/job_groups/backend/active_record/railtie.rb

```diff
@@ -0,0 +1,13 @@
+module Delayed
+  module JobGroups
+    module Backend
+      module ActiveRecord
+        class Railtie < ::Rails::Railtie
+          config.after_initialize do
+            Delayed::Backend::ActiveRecord::Job.send(:include, Delayed::JobGroups::JobExtensions)
```

Comment:
I know this code was copied from `lib/delayed_job_groups_plugin.rb` but while you're in here, can drop the `send` since this gem only support Ruby 2.5+ now?

```suggestion
            Delayed::Backend::ActiveRecord::Job.include(Delayed::JobGroups::JobExtensions)
```